run_all_tests.sh

```diff
@@ -0,0 +1,2 @@
+#!/usr/bin/env bash
+sbt -mem 2048 scalastyle compile coverage test coverageOff coverageReport
```

Comment:
Should we include it:test here too?

Reply:
@TheNickWilson - Have added it:test but had to remove scalastyle in order for it to run.